.gitignore

```diff
@@ -0,0 +1 @@
+.kitchen
```

Comment:
For now, I'm not using test kitchen in my workflow for roles, so can you please leave the .kitchen addition and .kitchen.yml file out of this PR?